src/MSBuild/app.amd64.config

```diff
@@ -60,6 +60,7 @@
         <dependentAssembly>
           <assemblyIdentity name="XamlBuildTask" culture="neutral" publicKeyToken="31bf3856ad364e35" />
           <bindingRedirect oldVersion="4.0.0.0" newVersion="16.0.0.0" />
+          <bindingRedirect oldVersion="15.0.0.0" newVersion="16.0.0.0" />
```

Comment:
I'd do the same thing here as you did for the other one.

Reply:
done.